### DISCUSSION TITLE
Creating DODa. Hub on Discord

### DISCUSSION THREAD

**Discussion by atw-9** [darija-open-dataset/dataset#172] [Ideas]
Hello,
I believe it would be a great idea to create a Discord server to facilitate sync and idea sharing among us. Also, it could serve as a place to unite the Moroccan NLP enthusiast community, what do you think?

**Comment by HaoES** [comment]
Hello @Boogey9 we are part of a discord server (not totally dedicated to DODa) I will send you the invite link by mail (the one you have on your profile) if you are interested, we also have a Whatsapp community if you are interested in joining you can give me your phone number by email to invite you. Thank you for your good work!
